tests/api.c

```diff
@@ -18498,7 +18498,11 @@ static int test_wc_AesGcmEncryptDecrypt(void)
         BAD_FUNC_ARG);
     ExpectIntEQ(wc_AesGcmDecrypt(&aes, dec, enc, sizeof(enc)/sizeof(byte), iv,
         sizeof(iv)/sizeof(byte), resultT, sizeof(resultT) + 1, a, sizeof(a)),
+    #if (defined(HAVE_FIPS) && FIPS_VERSION_LE(2,0) && defined(WOLFSSL_ARMASM))
+        AES_GCM_AUTH_E);
```

Comment:
Prefer the trailing `);` is outside the `#if` block.

Reply:
I might also be required you do this check outside `ExpectIntEQ` since that is also a macro.

Reply:
Done